rst_to_myst/parser.py

```diff
@@ -127,6 +128,13 @@ def apply(self):
             candidate.replace_self(front_matter)
 
 
+@lru_cache
```

Comment:
```suggestion
@lru_cache()
```

I believe this only changed in python 3.8, such that you can use `@lru_cache` instead of `@lru_cache()`